src/panels/config/script/ha-script-picker.ts

```diff
@@ -130,51 +125,57 @@ class HaScriptPicker extends LitElement {
         `,
       };
     }
-    columns.info = {
-      title: "",
-      label: this.hass.localize("ui.panel.config.script.picker.show_info"),
-      type: "icon-button",
-      template: (_info, script) => html`
-        <ha-icon-button
-          .script=${script}
-          @click=${this._showInfo}
-          .label=${this.hass.localize(
-            "ui.panel.config.script.picker.show_info"
-          )}
-          .path=${mdiInformationOutline}
-        ></ha-icon-button>
-      `,
-    };
-    columns.trace = {
-      title: "",
-      label: this.hass.localize("ui.panel.config.script.picker.dev_script"),
-      type: "icon-button",
-      template: (_info, script: any) => html`
-        <a href="/config/script/trace/${script.entity_id}">
-          <ha-icon-button
-            .label=${this.hass.localize(
-              "ui.panel.config.script.picker.dev_script"
-            )}
-            .path=${mdiHistory}
-          ></ha-icon-button>
-        </a>
-      `,
-    };
-    columns.edit = {
+
+    columns.actions = {
       title: "",
-      label: this.hass.localize("ui.panel.config.script.picker.edit_script"),
-      type: "icon-button",
-      template: (_info, script: any) => html`
-        <a href="/config/script/edit/${script.entity_id}">
-          <ha-icon-button
-            .label=${this.hass.localize(
-              "ui.panel.config.script.picker.edit_script"
-            )}
-            .path=${mdiPencil}
-          ></ha-icon-button>
-        </a>
-      `,
+      width: this.narrow ? undefined : "10%",
+      type: "overflow-menu",
+      template: (_: string, script: any) =>
+        html`
+          <ha-icon-overflow-menu
```

Comment:
We should stop using this component and move to just `ha-button-menu` as we no longer use the dynamic `narrow` function

Reply:
Let's do it in another PR though